cpp/abconv/CMakeLists.txt

```diff
@@ -81,6 +81,13 @@ elseif()
 	message(STATUS "EIC Afterburner: zlib not found. Compressed files support disabled.")
 endif()
 
```

Comment:
Would make sense to search for `find_package(HepMC3 REQUIRED COMPONENTS rootIO)` to test that user has that compiled in. From what I understand HEPMC3_LIBRARIES will already include HepMC3rootIO along with every other available library.

Another way is to `find_package(HepMC3 REQUIRED OPTIONAL_COMPONeNTS rootIO)` and use `HepMC3_rootIO_FOUND` and drop the `HEPMC_ROOTIO` option altogether.

Reply:
tbh, I just left it optional because the #ifdef was already there and I figured somebody must have had a reason for it. Considering the histograms that are created, root is needed anyway, but I just did some work in epic where `#ifdef ROOTIO`'s were added very deliberately so I was sensitized.

Reply:
I consider it to be fine. Agree with Kolja comment